python/ray/autoscaler/_private/commands.py

```diff
@@ -758,6 +750,61 @@ def get_or_create_head_node(config: Dict[str, Any],
         cli_logger.print("  {}", remote_shell_str.strip())
 
 
+def _should_create_new_head(head_node_id: Optional[str], new_launch_hash: str,
+                            new_head_node_type: str,
+                            provider: NodeProvider) -> bool:
+    """
+    Decides whether a new head node needs to be created.
```

Comment:
This should be at same line as top quote.
```suggestion
    """Decides whether a new head node needs to be created.
```

Reply:
I should probably use a doc string plugin.